manifests/i/ImageMagick/ImageMagick/7.1.0.36/ImageMagick.ImageMagick.locale.en-US.yaml

```
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: ImageMagick.ImageMagick
+PackageVersion: 7.1.0.36
+PackageLocale: en-US
+Publisher: ImageMagick Studio LLC
+PublisherUrl: https://github.com/ImageMagick/ImageMagick
+PublisherSupportUrl: https://github.com/ImageMagick/ImageMagick/issues
+# PrivacyUrl:
+Author: ImageMagick Studio LLC
+PackageName: ImageMagick
+PackageUrl: https://imagemagick.org
+License: Copyright (c) ImageMagick Studio LLC - derived Apache 2.0 license
```

```suggestion
License: Apache-2.0 Derivative
```
A copyright isn't a license and should not be included in the license field